### ISSUE TITLE
train loss with your pretrained model is 0

### ISSUE THREAD

**Issue by Zhang-O** [SakuraRiven/EAST#16]
Epoch is [1/600], mini-batch is [1/33], time consumption is 0.12597346, batch_loss is 0.00000000
Epoch is [1/600], mini-batch is [2/33], time consumption is 0.07961965, batch_loss is 0.00000000
Epoch is [1/600], mini-batch is [3/33], time consumption is 0.07560468, batch_loss is 0.00000000

**Comment by kavyajeetbora** [comment]
this is because torch.sum(gt_score) is < 1 
see the loss function it returns zero...if this above condition is true
I am trying to figure out why this is happening

**Comment by Adnan-annan** [comment]
any solution to this problem ?

**Comment by Adnan-annan** [comment]
does it affect the training result ?

**Comment by shihuangxiong** [comment]
> Epoch is [1/600], mini-batch is [1/33], time consumption is 0.12597346, batch_loss is 0.00000000
> Epoch is [1/600], mini-batch is [2/33], time consumption is 0.07961965, batch_loss is 0.00000000
> Epoch is [1/600], mini-batch is [3/33], time consumption is 0.07560468, batch_loss is 0.00000000

请问你这个实在Ubuntu系统运行还是win10??

**Comment by SakuraRiven** [comment]
sry for the late reply. `torch.sum(gt_score) is < 1` denotes there are no text pixels in this batch. A naive solution is enlarging batch_size to confirm the basic number of text pixels.
